src/main/kotlin/io/snyk/plugin/Severity.kt

```diff
@@ -0,0 +1,39 @@
+package io.snyk.plugin
+
+import com.intellij.util.ui.UIUtil
+import java.awt.Color
+
+class Severity {
+    companion object {
+        const val CRITICAL = "critical"
+        const val HIGH = "high"
+        const val MEDIUM = "medium"
+        const val LOW = "low"
+
+        fun getIndex(severity: String): Int =
+            when (severity) {
+                CRITICAL -> 4
+                HIGH -> 3
+                MEDIUM -> 2
+                LOW -> 1
+                else -> 0
+            }
+
+        fun toName(index: Int): String =
+            when (index) {
+                3 -> HIGH
+                2 -> MEDIUM
+                1 -> LOW
+                else -> LOW
```

Comment:
probbaly we should create special type of severity for Unknown index values... might be out of this PR scope

Reply:
Made a quick small fix for that use case: #113 @aldanchenko please have a look